package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "genesis-praxis",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+	"start": "node index.js"
```

Comment:
Different intentation on scriprs. Also, we don't need the `test` command if we don't have any unit tests